client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -283,6 +319,11 @@ export default {
       required: true
     },
 
+    statementFormDefinitions: {
```

Comment:
Do we know the possible properties? I'd find it useful to document them here

Reply:
Not yet, but I'll keep this in mind :+1: